### ISSUE TITLE
Parse & display stream info tags

### ISSUE THREAD

**Issue by h3298** [odioapp/odio#16]
Display embedded metadata (such as current program, song or station info) when the data is present in stream.  (Examples provided below that work in VLC)
https://www.videolan.org/vlc/

**Icecast/Shoutcast/etc**
http://24.120.173.242:8002/stream/2/

**iTunes media server format**
https://c9icy.prod.playlists.ihrhls.com/5260_icy
https://c3icy.prod.playlists.ihrhls.com/5060_icy

**Comment by spiral2k** [comment]
Hi @h3298, great feature!, I'll see if it can be done based on the data that i'm getting

**Comment by spiral2k** [comment]
@carcade can confirm that it will be in the next version, not working in all stations but in most of them.

**Comment by carcade** [comment]
> @carcade can confirm that it will be in the next version, not working in all stations but in most of them.

It will be great.

**Comment by malszycki** [comment]
Exactly. If song names would be visible in version 2.0 I would be most delighted!

**Comment by odioapp** [comment]
@malszycki it will, the next version will be ready for public beta in one month
